CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Misc:
 - [#1535](https://github.com/rails-api/active_model_serializers/pull/1535) Move the adapter and adapter folder to
   active_model_serializers folder and changes the module namespace. (@domitian @bf4)
 - [#1497](https://github.com/rails-api/active_model_serializers/pull/1497) Add JRuby-9000 to appveyor.yml(@corainchicago)
+- [#1496](https://github.com/rails-api/active_model_serializers/pull/1496) Run all branches against JRuby on CI
```

Comment:
@nadavshatz would you mind adding this to the top of the list and give yourself credit? `on CI (@nadavshatz)`. Thanks.